.github/workflows/ci.yml

```diff
@@ -24,8 +23,8 @@ jobs:
         os: [macos, ubuntu, windows]
 
         include:
-          - { type: shared, build: YES }
-          - { type: static, build: NO }
+          - {type: shared, build: YES}
+          - {type: static, build: NO}
```

Comment:
PR title is about cmake-lint, why is this file being touched anyway?

Reply:
yamlint ...